app/controllers/Error404.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * This class extends Controller class, this class show error page.
+ */
+class Error404 extends Controller {
+
+  /**
+   * This function show error page.
+   *
+   * @param  mixed $a
```

Comment:
Can we place provide some meaningful names to the variables that denoted the purpose of it ?